### DISCUSSION TITLE
Welcome to autotask-client Discussions!

### DISCUSSION THREAD

**Discussion by aidan-casey** [Anteris-Dev/autotask-client#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by optmsp** [comment]
I am using autotask-client in a pet project for a SaaS tool. It's been very helpful, especially coming from the opendns/autotask client, and has saved me a lot of time. Do you have a recurring donation feature like other open source projects sometimes do?
